src/mscorlib/corefx/System/Globalization/CompareInfo.Unix.cs

```diff
@@ -312,5 +312,12 @@ private static byte[] GetNullTerminatedUtf8String(string s)
 
             return buffer;
         }
+        
+        private bool CanUseOrdinal(string source, string target, CompareOptions options)
+        {
+            return m_isAsciiEqualityOrdinal && CanUseAsciiOrdinalForOptions(options) && 
+                   source.IsAscii() && !source.IsFastSort() && 
+                   target.IsAscii() && !target.IsFastSort();
```

Comment:
What does IsFastSort actually do?  I'm wondering if ```!IsFastSort()``` is a pure subset of ```IsAscii()``` such that we could eliminate the former check and just do the latter?  It'd also be good to get some perf measurements on this, e.g. to ensure that the extra call doesn't somehow tip the scales and eliminate the value of the optimization.